src/Neo/SmartContract/Native/NeoToken.cs

```diff
@@ -193,7 +193,10 @@ internal override ContractTask InitializeAsync(ApplicationEngine engine, Hardfor
                 engine.SnapshotCache.Add(CreateStorageKey(Prefix_Committee), new StorageItem(cachedCommittee));
                 engine.SnapshotCache.Add(_votersCount, new StorageItem(System.Array.Empty<byte>()));
                 engine.SnapshotCache.Add(CreateStorageKey(Prefix_GasPerBlock).AddBigEndian(0u), new StorageItem(5 * GAS.Factor));
```

Comment:
ping @shargon

Reply:
I think that maybe it should be enabled with 1 instead of 5 after certain height.
So the hardfork will be triggered by modification the `SnapshotCache` with`new StorageItem(5 * GAS.Factor)`;

Reply:
This should be moved to `config.json`?

Reply:
We can't change it without affecting post-genesis state. And I see zero reason to do this, mainnet/testnet don't care, this will be adjusted by the committee, private networks are fine with whatever default and they can also adjust it if needed.